projects/admin/src/app/record/detail-view/document-detail-view/holding/default-holding-item/default-holding-item.component.html

```diff
@@ -14,69 +14,93 @@
+  <!-- METADATA -->
+  <div class="row">
+    <!-- Barcodes -->
+    <div class="col-3 pl-5 font-weight-bold label-title" translate>Barcode</div>
+    <div class="col-9">
       <a [routerLink]="['/records', 'items', 'detail', item.metadata.pid]" name="barcode">
         {{ item.metadata.barcode }}
       </a>
     </div>
-    <div class="col-sm-2" name="status">{{ item.metadata.status | translate }}</div>
-    <div class="col-sm-3" name="issue">{{ item.metadata.enumerationAndChronology }}</div>
-    <div class="col-sm-2" name="call-number">
+    <!-- Unit -->
+    <ng-container *ngIf="item.metadata.enumerationAndChronology">
+      <div class="col-3 pl-5 font-weight-bold label-title" translate>Unit</div>
+      <div class="col-9">
+        {{ item.metadata.enumerationAndChronology }}
+      </div>
+    </ng-container>
+    <!-- Call numbers -->
+    <div class="col-3 pl-5 font-weight-bold label-title" translate>Call number</div>
+    <div class="col-9">
       <shared-inherited-call-number [item]="item"></shared-inherited-call-number>
     </div>
-    <div class="col-sm-2 text-right" name="buttons">
-      <button *ngIf="permissions.canRequest && permissions.canRequest.can; else notRequest"
-              type="button" class="btn btn-sm btn-outline-primary " (click)="addRequest(item.metadata.pid)"
-              title="{{ 'Item request' | translate}}"
+    <!-- Status -->
+    <div class="col-3 pl-5 font-weight-bold label-title" translate>Status</div>
+    <div class="col-9">
+      <i class="mr-1 fa fa-circle text-{{ item.metadata.availability.available ? 'success' : 'danger' }}"></i>
+      <ng-container [ngSwitch]="item.metadata.status">
+        <span *ngSwitchCase="'on_loan'">
+          {{ 'due until' | translate }}
+          {{ item.metadata.availability.due_date | dateTranslate :'shortDate' }}
+        </span>
+        <span *ngSwitchDefault>
+          {{ item.metadata.availability.display_text | getTranslatedLabel }}
+        </span>
+      </ng-container>
+      <ng-container *ngIf="item.metadata.availability.request > 0">
+        ({{ item.metadata.availability.request }} {{ (item.metadata.availability.request > 1 ? 'requests' : 'request') | translate }})
+      </ng-container>
+    </div>
+    <!-- URL -->
+    <ng-container *ngIf="item.metadata.url">
+      <div class="col-3 pl-5 font-weight-bold label-title" translate>Online access</div>
+      <div class="col-9"><a href="{{ item.metadata.url }}">{{ item.metadata.url }}</a></div>
+    </ng-container>
+  </div>
+  <admin-holding-item-note [note]="note" *ngFor="let note of item.metadata.notes"></admin-holding-item-note>
+  <admin-holding-item-temporary-item-type [record]="item"></admin-holding-item-temporary-item-type>
+  <admin-holding-item-in-collection [itemPid]="item.metadata.pid"></admin-holding-item-in-collection>
+  <!-- ACTIONS BUTTONS -->
+  <div class="position-absolute" [style]="{'top': 0, 'right':0 }" name="buttons">
```

Comment:
style seems not really clean here. Can you use sticky or other bootstrap property or at least a css class.